README.md

```diff
@@ -1,2 +1,13 @@
 # bucketlist
-This bucketlist application is built with the flask framework, python 3.6, sqlalchemy orm, bootstrap and a postgres dbms as it's backend database. In this application users can browse through the public wishes, create accounts for them to create their own wishes and manage their personal acccounts and wishes.
+This bucketlist application is built with the flask framework, python 3.6, sqlalchemy orm, bootstrap and a postgres dbms as it's backend database. 
+This application allows users do the following:
+1. Browse through the public wishes.
+2. Create accounts for themselves so as to create their own wishes.
+3. Manage their personal acccounts and wishes.
+
+Installation Guide:
+1. Create a virtual environment then pip install all the packege in the requirements.txt file.
+2. Create a database migration by running: python manage.py initdb.
+
+Application Stage:
+This application is currently at the backend stage. All routes are working but whats remaining is logic implementation.
```

Comment:
Good work on making your README more elaborate. There is a good template on a Github README that I recommend at: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2